content/devs.json

```diff
@@ -215,6 +215,39 @@
         "url": "mailto:jayemndz@gmail.com"
       }
     ]
+  },
+  {
+    "avatar": "https://i.ibb.co/qD9PkhH/me-min.jpg",
```

Comment:
Can you optimize the image? It's 96 kB. The avatar must below 80 kB.

Reply:
hi @jromest , can you check again, I think I already resolved the conflict, and updated the avatar url.